src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -142,11 +142,26 @@ export class SbbFormField implements ComponentInterface {
     this._namedSlots = queryNamedSlotState(this._element, this._namedSlots, event.detail);
   }
 
+  @Listen('will-open')
```

Comment:
I'm still not very happy about the coupling to the tooltip, but you probably did not found "better" solutions?

Reply:
detection via trigger is maybe a solution, but would need mutationobserver :/ and finding all possible triggers can be expensive. :has() selector is still not default available in firefox, but would be a nice option, e.g. something like `:host(:has([aria-expanded]))`, but maybe we can add a todo, as. during summer :has() should be available